lib/project_types/extension/models/specification_handlers/beacon_extension.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+require "base64"
+require "fileutils"
+require "json"
+
+module Extension
+  module Models
+    module SpecificationHandlers
+      class BeaconExtension < Default
+        SCRIPT_FILE = "beacon_extension.js"
+        CONFIG_FILE = "config.json"
+
```

Comment:
@sandor-shopify just to clarify:

- `config.json` is used to tell us whether the partner provided JS should be sandboxed or not. i.e. it is a JSON object that is expected to only contain one key for now, i.e.

```
{ sandbox: "true" }
```

but we will need an additional YAML file to hold the merchant config schema as we discussed in KD2, but that will be done as a followup to this PR?

Reply:
Look at my comment above - perhaps we should combine the schema and this `config.json` into a single `extension.config.yml` for simplicity